test/00/t0003a.sh

```diff
@@ -38,7 +38,10 @@ fubar
 if test $? -ne 0; then no_result; fi
 
 srec_cat test.in -intel -o test.out -motorola -header HDR > LOG 2>&1
-if test $? -ne 0; then cat LOG; fail; fi
+if test $? -ne 0; then
```

Comment:
Curious about why proposing a change to multi-line here while leaving line 38 unchanged. I prefer the single line for this kind of boilerplate but happy to be convinced otherwise. (more instances below).

Reply:
This was automatically done by `shfmt`. Couldn't find any hint in the [Google ShellGuide](https://google.github.io/styleguide/shellguide.html).

But I assume the rationale is, that multi-line shall be used if more than 1 command is used.

Reply:
That makes sense. I can't find any way to configure this in `shfmt` either. On balance, I really like being able to get consistent formatting so let's just keep the change as is.